src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,144 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        displayWelcomeMsg();
+
+        Task[] array = new Task[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        boolean looper = true;
+
+        while(looper) {
+            line = in.nextLine();
+            String lineCmd = line.trim().split(" ")[0];
+            switch(lineCmd){
+            case "bye":
+                displayByeMsg();
+                looper = false;
+                break;
+            case "list":
+                cmdList(array);
+                break;
+            case "done":
+                cmdDone(array, line);
+                break;
+            case "todo":
+                cmdTodo(array, line);
+                break;
+            case "deadline":
+                cmdDeadline(array, line);
+                break;
+            case "event":
+                cmdEvent(array, line);
+                break;
+            default:
+                System.out.println("Invalid command.");
+            } //end switch statement
+        } //end while loop
+    }
+
+    /* method for event command */
```

Comment:
Method comment not really necessary since the method name itself is self explanatory :)